hack/tools/catalogs/lib/bundle.sh

```diff
@@ -0,0 +1,194 @@
+# Library of functions for interacting with bundles and their manifests
+
+# SCRIPT_ROOT is the root of the script
+source "$(dirname "${BASH_SOURCE[0]}")/hash.sh"
+
+# Given package and version grabs the bundle image from stdin FBC stream
+function get-bundle-image(){
```

Comment:
nit: You don't need to put both the "function" keyword and "()".  The latter is in good form, the former is outdated.

Reply:
Fixed up ^^